### PR TITLE
Adds ability to upload test fixture via YAML file

### PR DESCRIPTION
**Pull request by dylankelly**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-

### Changed

1.  Adds createNodeFromYAML function to TideAdmin in ripple-test-tools
1.  Clean up and organise tests in vic-gov-au
1.  Create-ripple-app - adds feature to add e2e tests for enabled modules
1.  Move common test step definitions to ripple-test-tools
1.  Update event test to use YAML file upload

### CODE REVIEW

File: packages/create-ripple-app/template/_.env [hunk 1]
```diff
@@ -20,12 +20,10 @@ SEARCH_AUTH_USERNAME=<%= searchusername %>
 SEARCH_AUTH_PASSWORD=<%= searchpassword %>
 
 # Disable basic auth in local env.
-BASIC_AUTH=1
+BASIC_AUTH=0
```
Comment (tim-yao) [inline packages/create-ripple-app/template/_.env]: This need to be 1 as lagoon still use it for PR env, unless we have playbook setup for new project env without commit .env file.

File: packages/create-ripple-app/template/_example/components/examples/Button.vue [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<template>
```
Comment (tim-yao) [inline packages/create-ripple-app/template/_example/components/examples/Button.vue]: Not sure where to finally add these examples in the future, create-app or example site. Can you please put some thoughts here?

**Comment by tim-yao** [comment]
@dylankelly Just a high level CR, a few comments for you to have a look. Cheers!

File: packages/create-ripple-app/template/_.env (hunk 1 above)
Comment (dylankelly) [reply packages/create-ripple-app/template/_.env]: fixed

File: .eslintignore [hunk 3]
```diff
@@ -7,6 +7,7 @@ node_modules
 packages/ripple-nuxt-tide/templates/*
 packages/ripple-nuxt-tide/**/templates/*
 packages/ripple-nuxt-ui/**/templates/*
+packages/create-ripple-app/template/nuxt/*
```
[the remark this answers is not in the document]
Comment (dylankelly) [reply .eslintignore]: fixed

File: packages/create-ripple-app/template/gitignore [hunk 4]
```diff
@@ -7,5 +7,7 @@ npm-debug.log
 dist
 coverage
 
+# tests
+coverage
```
[the remark this answers is not in the document]
Comment (dylankelly) [reply packages/create-ripple-app/template/gitignore]: fixed

File: packages/create-ripple-app/saofile.js [hunk 5]
```diff
@@ -10,14 +10,13 @@ const TIDE_MODULES = [
...
-  { value: 'grant', name: 'Grant', default: true },
+  { value: 'grant', name: 'Grant', default: false },
   { value: 'gtm', name: 'Google Tag Manager', default: true },
   { value: 'profile', name: 'Profile', default: true },
   { value: 'media', name: 'Media', default: true },
   { value: 'webform', name: 'Webform', default: true },
   { value: 'search', name: 'Search', default: true },
   { value: 'publication', name: 'Publication', default: false },
-  { value: 'monsido', name: 'Monsido', default: true },
   {
```
[the remark this answers is not in the document]
Comment (dylankelly) [reply packages/create-ripple-app/saofile.js]: removed

File: packages/create-ripple-app/template/_example/components/examples/Button.vue (hunk 2 above)
Comment (dylankelly) [reply packages/create-ripple-app/template/_example/components/examples/Button.vue]: I've added a script which will copy example content from the reference site, update process would be to create a new site with create-ripple-app, do changes and then run script to update CRA

Comment (tim-yao) [reply packages/create-ripple-app/template/_example/components/examples/Button.vue]: Hi @dylankelly How would you suggest we run this `update.sh`? Manually before each release?

File: examples/vic-gov-au/package.json [hunk 6]
```diff
@@ -27,6 +33,11 @@
     "@babel/plugin-transform-runtime": "^7.4.4",
     "@babel/preset-env": "^7.5.0",
     "@babel/runtime": "^7.5.1",
+    "@dpc-sdp/ripple-test-tools": "1.0.0-beta.12",
```
Comment (tim-yao) [inline examples/vic-gov-au/package.json]: We need update this to `^1.0.0`

Comment (dylankelly) [reply examples/vic-gov-au/package.json]: I've updated this

**Pull request merged**